sos/report/plugins/scsi.py

```diff
@@ -39,6 +40,14 @@ def setup(self):
             "/sys/class/scsi_generic"
         ])
 
+        scsi_types = ["enclosu"]
+        result = self.collect_cmd_output('lsscsi -g')
```

Comment:
Ack, but remove the other call to `lsscsi -g` on line 59 so we don't duplicate collections.

Reply:
Removed the duplicate call to lsscsi -g

Reply:
Hi @TurboTurtle , could you please review again?